Homework7/Group.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Homework7
+{
+    class Group
+    {
+        public string GroupName { get; }
```

Comment:
Do you really require Group and unit names? You can use indexes to detect winners or something already what already exist.

Reply:
not required, added just for visual clarity when displaying results
fixed